NetCord/Rest/RestGuild.cs

```diff
@@ -102,4 +62,274 @@ int GetHighestRolePosition(PartialGuildUser user)
+    /// </summary>
+    public string PreferredLocale => _jsonModel.PreferredLocale;
+
+    /// <summary>
+    /// The ID of the channel where admins and moderators of community guilds receive notices from Discord.
+    /// </summary>
+    public ulong? PublicUpdatesChannelId => _jsonModel.PublicUpdatesChannelId;
+
+    /// <summary>
+    /// The maximum amount of users in a video channel.
+    /// </summary>
+    public int? MaxVideoChannelUsers => _jsonModel.MaxVideoChannelUsers;
+
+    /// <summary>
+    /// The maximum amount of users in a stage video channel.
+    /// </summary>
+    public int? MaxStageVideoChannelUsers => _jsonModel.MaxStageVideoChannelUsers;
+
+    /// <summary>
+    /// The approximate number of <see cref="GuildUser"/>s in the <see cref="RestGuild"/>.
+    /// </summary>
+    /// <remarks>
+    /// Only available in objects returned from <see cref="RestClient.GetGuildAsync(ulong, bool, RestRequestProperties?)"/> and <see cref="RestClient.GetCurrentUserGuildsAsync(GuildsPaginationProperties?, RestRequestProperties?)"/>, where <c>withCounts</c> is true.
+    /// </remarks>
+    public int? ApproximateUserCount => _jsonModel.ApproximateUserCount;
+
+    /// <summary>
+    /// Approximate number of non-offline <see cref="GuildUser"/>s in the <see cref="RestGuild"/>.
+    /// </summary>
+    /// <remarks>
+    /// Only available in objects returned from <see cref="RestClient.GetGuildAsync(ulong, bool, RestRequestProperties?)"/> and <see cref="RestClient.GetCurrentUserGuildsAsync(GuildsPaginationProperties?, RestRequestProperties?)"/>, where <c>withCounts</c> is true.
+    /// </remarks>
+    public int? ApproximatePresenceCount => _jsonModel.ApproximatePresenceCount;
+
+    /// <summary>
+    /// The welcome screen shown to new members, returned in an invite's <see cref="RestGuild"/> object.
+    /// </summary>
+    public GuildWelcomeScreen? WelcomeScreen { get; }
+
+    /// <summary>
+    /// The <see cref="RestGuild"/>'s set NSFW level.
+    /// </summary>
+    public NsfwLevel NsfwLevel => _jsonModel.NsfwLevel;
+
+    /// <summary>
+    /// A dictionary of <see cref="GuildSticker"/> objects indexed by their IDs, representing the <see cref="RestGuild"/>'s custom stickers.
+    /// </summary>
+    public ImmutableDictionary<ulong, GuildSticker> Stickers { get; set; }
+
+    /// <summary>
+    /// Whether the <see cref="RestGuild"/> has the boost progress bar enabled.
+    /// </summary>
+    public bool PremiumProgressBarEnabled => _jsonModel.PremiumProgressBarEnabled;
+
+    /// <summary>
+    /// The ID of the channel where admins and moderators of community guilds receive safety alerts from Discord.
+    /// </summary>
+    public ulong? SafetyAlertsChannelId => _jsonModel.SafetyAlertsChannelId;
+
+    /// <summary>
+    /// The ID used for @everyone mentions.
```

Comment:
🤔

Reply:
Not sure what I was doing here, but corrected it.